src/Tribe/RSVP.php

```diff
@@ -312,8 +312,15 @@ public function generate_tickets( ) {
 				update_post_meta( $attendee_id, $this->order_key, $order_id );
 				update_post_meta( $attendee_id, $this->full_name, $attendee_full_name );
 				update_post_meta( $attendee_id, $this->email, $attendee_email );
+				
+				do_action ( 'tickets_rsvp_tickets_created', $attendee_id, $event_id, $product_id );
 			}
+			
+			do_action( 'tickets_rsvp_tickets_generated_for_product', $product_id, $order_id, $qty );
```

Comment:
Can this action be changed to: `event_tickets_rsvp_tickets_generated_for_product`?